src/aws-cpp-sdk-core/include/aws/core/utils/stream/SimpleStreamBuf.h

```diff
@@ -42,6 +42,12 @@ namespace Stream
 
             void swap(SimpleStreamBuf& rhs);
 
+            /// returns pointer to the underlying data buffer.
+            char* get_buffer_data();
```

Comment:
thanks for your change!

Could you please use the styling convention used in this sdk? (i.e. the PascalCase, and call it `GetBufferData()`?)

Unless you wanted to make it compatible with some std:: stringbuf interface like [`basic_stringbuf`](https://en.cppreference.com/w/cpp/io/basic_stringbuf) (but I could not find if any std:: class has such method).